public/styles/components/_footer.scss

```diff
@@ -151,9 +151,15 @@
         .bsds-footer-social-media-icon {
           width: 100%;
           height: space(2x);
+        }
+
+        .bsds-link-subtle {
+          &:visited {
+            color: $text-color-primary;
```

Comment:
can we look at where we're using text color abstracted vars vs regular color names since this is getting applied to icons?